app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,18 @@
+class DishIngredientsController < ApplicationController
+
+  def create
+    DishIngredient.create!(strong_params)
+    redirect_to dish_path(strong_params[:dish_id])
+  end
+
+  def destroy
+    DishIngredient.destroy(strong_params[:id])
```

Comment:
Nice work getting to this extension! IF it's tricky to get the dish_ingredient ID, you can also use `find_by` and pass both the ingredient_id and the dish_id to find the exact DishIngredient object. Then, you can destroy that object